Base/res/ladybird/inspector.css

```diff
@@ -147,11 +147,11 @@ html {
     color: var(--tab-button-active-color);
 }
 
-.tab-controls button + button {
+.tab-controls button+button {
     border-left: 1px solid var(--tab-button-border);
 }
 
-details > :not(:first-child) {
+details> :not(:first-child) {
```

Comment:
Please avoid making unrelated formatting changes.

Reply:
Sorry, I had autoformatter that changed it without me noticing. I'll revert this change.

Reply:
Fixed it!